Dockerfile

```diff
@@ -71,12 +68,8 @@ RUN ln -s /toolchain/bin/pwd /bin/pwd && \
     cp /tmp/syslinux/bios/extlinux/extlinux /rootfs/bin && \
     cp /tmp/syslinux/efi64/mbr/gptmbr.bin /rootfs/share
 # golang
-ENV GOROOT /toolchain/usr/local/go
-ENV GOPATH /toolchain/go
-COPY --from=golang-musl /usr/local/go ${GOROOT}
-ENV PATH ${PATH}:${GOROOT}/bin
+ENV GOPATH /toolchain/gopath
```

Comment:
essentially this is pretty much dummy, but having no `$GOPATH` means some packages won't build, as outside `$GOPATH` packages start automatically using modules, and this breaks docker-related stuff (containerd), as there's dependency case sensitivity hell with logrus package (https://github.com/golang/go/issues/28489)